### PR TITLE
translating different small pages

### PR DESCRIPTION
**Pull request by Karol-Maximus**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -397,6 +401,11 @@ en:
         filled_6: "The survey Nº6 has been done"
         reg_fee: "Pay the registration fee"
         book_appointment: "Book an appointment"
+      studies:
+        available_studies: "Available Studies"
+        study_duration: "Study duration "
+        months: " months"
+        register: "Register"
```
Comment (wolak88) [inline config/locales/en.yml:408]: I think I know what you want to achieve here, but let's add those whitespaces in the view file.

File: config/locales/pl.yml [hunk 2]
```diff
@@ -355,6 +359,11 @@ pl:
         filled_6: "Ankieta nr 6 wypełniona"
         reg_fee: "Wnieś opłatę wstępną"
         book_appointment: "Umów się na wizytę"
+      studies:
+        available_studies: "Dostępne Badania"
+        study_duration: "Okres "
+        months: " miesięcy"
+        register: "Zapisz się"
```
Comment (wolak88) [inline config/locales/pl.yml:366]: I think I know what you want to achieve here, but let's add those whitespaces in the view file.

**Review by wolak88 — CHANGES_REQUESTED** [review]
Just some nitpicking

File: app/views/participant_portal/partials/_questionscol.html.haml [hunk 3]
```diff
@@ -4,10 +4,10 @@
 
       / Title
       .span
-        .text-sm.text-gray-600.mb-3.text-center.capitalize.md:text-2xl #{local_assigns[:s].survey.study.title}
```
Comment (roxkon) [inline app/views/participant_portal/partials/_questionscol.html.haml:7]: I think that Study Title should be capitalized

File: config/locales/pl.yml [hunk 4]
```diff
@@ -312,7 +310,13 @@ pl:
         dry_10: "Susz: 10g miesięcznie, Olejek: 5g miesięcznie" 
         dry_20: "Susz: 20g miesięcznie, Olejek: 10g miesięcznie"
         dry_30: "Susz: 20g miesięcznie, Olejek: 10g miesięcznie"
```
Comment (roxkon) [inline config/locales/pl.yml:312]: Is it right or is it wrong? I think dry_30 should be "Susz: 30g miesięcznie, Olejki: 15g miesięcznie"

Comment (Karol-Maximus) [reply config/locales/pl.yml]: Nope. it is changed now.

File: app/views/participant_portal/partials/_questionscol.html.haml (hunk 3 above)
Comment (Karol-Maximus) [reply app/views/participant_portal/partials/_questionscol.html.haml:7]: It is a survey title that's why i removed the capitalizing

File: config/locales/pl.yml (hunk 2 above)
Comment (Karol-Maximus) [reply config/locales/pl.yml]: done

File: config/locales/en.yml (hunk 1 above)
Comment (Karol-Maximus) [reply config/locales/en.yml]: done

**Review by wolak88 — APPROVED** [review]

**Pull request merged**